exception_reports/decorators.py

```diff
@@ -41,10 +41,20 @@ def _exception_reports(func, *args, **kwargs):
             filename = gen_error_filename(extension='html')
             report_location = storage_backend.write(filename, data)
             setattr(e, 'report', report_location)
+
+            ExceptionType = type(e)
+
+            def my_str(self):
+                m = ExceptionType.__str__(self)
+                return f'{m} [report:{report_location}]'
+
+            NewExceptionType = type(ExceptionType.__name__ + "_Override", (ExceptionType,), {'__str__': my_str})
```

Comment:
Visually it seems pretty annoying to see `_Override` appended to my exception whenever using this library—I get that it could aid in debugging if something goes wrong with this logic, but ideally this library should work and users can trust that it will work, and not need to be reminded everytime they see an exception printed to the console that we're overriding the exception to append a URL to exception report.